test/wake_word/wake_word_test.py

```diff
@@ -89,6 +89,8 @@ def unmute(self):
     def close(self):
         self.stream.close()
 
+    def duration_to_bytes(self, ww_duration):
+        return int(ww_duration*self.SAMPLE_RATE*self.SAMPLE_WIDTH)
```

Comment:
Maybe add spaces around the `*` to match the general code style

There is a blank line missing here as well. (There should be 2 blank lines between classes / global functions, 1 blank line between methods)